dozer-tests/src/sqllogictest/suits/join.test

```diff
@@ -39,9 +39,6 @@ CREATE TABLE film_actor (
 statement ok
 INSERT INTO actor(actor_id,first_name, last_name, last_update) VALUES (1, 'penelope', 'guiness', '2020-02-15 09:34:33+00');
 
-statement ok
```

Comment:
I deleted this insert, the old join test doesn't have this insert.

`map_operation_to_sql` in `mapper` is invalid, we should find a better way to get dozer's output in the future. @mediuminvader